### PR TITLE
Adds notebook-controller manifest

### PR DESCRIPTION
**Pull request by swiftdiaries**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/9)

### CODE REVIEW

File: notebook-controller/base/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+namePrefix: notebooks-controller-
+
+resources:
```
Comment (lluunn) [inline notebook-controller/base/kustomization.yaml]: is it common pattern to have one resource per file?

Comment (swiftdiaries) [reply notebook-controller/base/kustomization.yaml]: Yeah, one set of resources per component

**Comment by swiftdiaries** [comment]
/cc @lluunn @jlewi 
I think this is good to be merged

**Pull request merged**